corehq/apps/userreports/tests/test_pillow.py

```diff
@@ -510,8 +510,8 @@ def test_reuse_cache(self):
         self._test_reuse_cache()
 
     def test_reuse_cache_chunked(self):
-        pillow1 = get_kafka_ucr_pillow(topics=['case-sql'], processor_chunk_size=100)
-        pillow2 = get_kafka_ucr_pillow(topics=['case-sql'], processor_chunk_size=100)
+        pillow1 = get_case_pillow(topics=['case-sql'], processor_chunk_size=100, ucr_configs=self.configs[:1])
+        pillow2 = get_case_pillow(topics=['case-sql'], processor_chunk_size=100, ucr_configs=self.configs)
```

Comment:
why are the configs different in these 2 pillows?

Reply:
This is just a refactor of old code. The test is about using two pillows with different configs and testing them. Earlier code was relying on extra `bootstrap` call to attach ucr configs, I have updated it so that configs are directly passed at initialisation and attached.